contracts/lib/OrderCombiner.sol

```diff
@@ -625,6 +622,9 @@ contract OrderCombiner is OrderFulfiller, FulfillmentApplier {
         // Put ether value supplied by the caller on the stack.
         uint256 etherRemaining = msg.value;
 
+        // Retrieve the length of the executions array and place on stack.
+        uint256 totalExecutions = executions.length;
+
```

Comment:
I put this above the accumulator variable based off the accumulator variable comment. Let me know if im misreading this. currently running tests locally to see if it will pass while the variable is allocated after the accumulator variable.

Reply:
oooh that only corrupts new memory declared after the accumulator is instantiated; you can / should just use it right before the iterator!

Reply:
i.e. executions has already been allocated in memory

Reply:
Ooooooh got it. that makes sense thanks for explaining.